x-pack/test/functional/services/cases/navigation.ts

```diff
@@ -14,7 +14,7 @@ export function CasesNavigationProvider({ getPageObject, getService }: FtrProvid
   return {
     async navigateToApp(app: string = 'cases', appSelector: string = 'cases-app') {
       await common.navigateToApp(app);
-      await testSubjects.existOrFail(appSelector, { timeout: 2000 });
+      await testSubjects.existOrFail(appSelector);
```

Comment:
I assume dropping the explicit timeout falls back to some default (that's not 2s)?

Reply:
Oh I see it's a config setting: https://github.com/elastic/kibana/blob/0ab845a2e15fd5b659c9fd801642b324eb8a9418/test/functional/services/common/test_subjects.ts#L32

Reply:
Yeah, the default timeout is 2 minutes (based on the config, but described in the description of the method), which is a long time but the best practice is really to not go around customizing the timeouts unless there's a good reason for it in the tests. I don't think this meets that threshold.

Reply:
Gotcha, I hadn't seen the comments and was tracing the config values in @kbn/test, but now that I've read those comments it's all clear. Thanks!